glide.yaml

```diff
@@ -12,6 +12,7 @@ import:
 - package: github.com/btcsuite/websocket
 - package: github.com/decred/bitset
 - package: github.com/decred/dcrd
+  version: eda86f5267b747f095da4235cfb10fa361b5f769
```

Comment:
We should be tracking master so this is unneeded.  Only have to commit the lockfile.